qemu/tests/blockdev_mirror_qemuio_ready_job.py

```diff
@@ -0,0 +1,61 @@
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import qemu_storage
+
+from provider.blockdev_mirror_nowait import BlockdevMirrorNowaitTest
+from provider.job_utils import get_event_by_condition
+
+
+class BlockdevMirrorQemuioReadyjob(BlockdevMirrorNowaitTest):
+    """
+    Qemuio source image when mirror job reach ready status
+    """
+
+    def qemuio_source_image(self):
+        tag = self._source_images[0]
+        image_params = self.params.object_params(tag)
+        img_format = image_params.get("image_format")
+        filename = qemu_storage.get_image_repr(tag, image_params,
+                                               data_dir.get_data_dir(),
+                                               'filename')
+        qemuio_cmd = self.params.get("qemuio_cmd") % (img_format, filename)
+        try:
+            process.run(qemuio_cmd, shell=True)
+        except process.CmdError as e:
```

Comment:
Still, we need a else to handle the 'unexpected success'

Reply:
Have update, thanks @zhencliu